frontend/src/components/button.module.css

```diff
@@ -3,22 +3,39 @@
   border-radius: 20px;
 }
 
+.button_container:not(:has(.text)) {
```

Comment:
I don't think there's a `.text` class defined so this always applies? 
If so, could this just be merged into above `.button_container`?